server/routes/users/updateUserPassword.route.js

```diff
@@ -0,0 +1,51 @@
+var UserModel = require('../../models/user.model');
+var ValidationData = require('../../helper/validationIncomingData');
+var PasswordHasher = require('../../helper/passwordHasher');
+
+var updateUserPasswordRoute = function (req, res){
+
+    var fieldToValidate = ["id"];
+    var errorMessage = ValidationData(fieldToValidate, req.params);
+
+    //check passwords are equall
+    if (req.body.password !== req.body.password_confirmation){
+        errorMessage+= "Passwords doesn't match";
+    }
+
+    if(errorMessage != "") {
```

Comment:
Do not check string for returning the result. Return this if the passwords doesn't match

Reply:
I'm checking the string here because errorMessage can be modified in both line 8 and line 12. It's incorrect to do so? Otherwise, I'd have to check errorMessage twice.

Reply:
Makes sense, leave as it is